### PR TITLE
Multiple instances of IProvideConfiguration for the same configuration section

### PR DESCRIPTION
**Pull request by SeanFeldman**
Note added to clarify core issue https://github.com/Particular/NServiceBus/issues/3649

@Particular/docs-maintainers please review

/cc @timbussmann

### CODE REVIEW

File: nservicebus/hosting/custom-configuration-providers.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ snippet: CustomConfigProvider
 
 Adding the code above is enough since NServiceBus will automatically use it found in the types scanned.
 
+NOTE: if multiple instances of `IProvideConfiguration<T>` for the same configuration sections `T` are found, only one instance will be used.
```
Comment (SimonCropp) [inline nservicebus/hosting/custom-configuration-providers.md]: > only one instance will be used.
 
which one? 

change "if multiple" to "If multiple"

Comment (SimonCropp) [inline nservicebus/hosting/custom-configuration-providers.md]: this sounds like we are documenting a bug

Comment (indualagarsamy) [inline nservicebus/hosting/custom-configuration-providers.md]: @SimonCropp - good point. Can we document this for the time being and open an issue in the core to address this?

Comment (SimonCropp) [inline nservicebus/hosting/custom-configuration-providers.md]: i prefer not to document bugs

Comment (indualagarsamy) [inline nservicebus/hosting/custom-configuration-providers.md]: @SimonCropp - see https://github.com/Particular/NServiceBus/issues/3649#issuecomment-204355499

**Comment by indualagarsamy** [comment]
@SeanFeldman @timbussmann - I've raised an issue in the core https://github.com/Particular/NServiceBus/issues/3660.

While I agree that we need to document the behavior, having an issue open does the same and allows people to comment on it. Talking it through with @SimonCropp, I feel the same way, i.e. we are documenting the bug. So, I am closing this PR.

**Pull request closed without merging**